### PR TITLE
remove arbitrary stability constant in fisher information

### PR DESCRIPTION
**Pull request by avati**
Also minor edits in line search

### CODE REVIEW

File: ngboost/ngboost.py [hunk 1]
```diff
@@ -159,11 +159,9 @@ def line_search(self, resids, start, Y, sample_weight=None, scale_init=1):
...
             loss = D.total_score(Y, sample_weight)
             norm = np.mean(np.linalg.norm(scaled_resids, axis=1))
-            if (
-                np.isfinite(loss)
-                and (loss < loss_init or norm < self.tol)
-                and np.linalg.norm(scaled_resids, axis=1).mean() < 5.0
-            ):
+            if norm < self.tol:
+                break
+            if np.isfinite(loss) and loss < loss_init:
```
Comment (alejandroschuler) [inline ngboost/ngboost.py:164]: hey @avati could you explain this change a little bit?

Comment (avati) [reply ngboost/ngboost.py:164]: I just broke down a complex conditional into two parts for easier readability, and also removed one of the conditions which doesn't make a lot of sense (not sure it should've been there in the first place).

**Comment by avati** [comment]
Version bumped up to 0.2.3dev

**Comment by tonyduan** [comment]
lgtm!

**Pull request merged**